<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileConversionClass</class>
 <widget class="QMainWindow" name="FileConversionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileConversion</string>
  </property>
  <property name="windowIcon">
   <iconset resource="fileconversion.qrc">
    <normaloff>:/FileConversion/Resources/exe.ico</normaloff>:/FileConversion/Resources/exe.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_13">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_12">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Frame</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="0">
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>TotalFrame:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>CurrentFrame:</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QPushButton" name="backButton">
                 <property name="text">
                  <string>Back</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QPushButton" name="nextButton">
                 <property name="text">
                  <string>Next</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="totalframelineEdit">
                 <property name="text">
                  <string>0</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="currentlineEdit">
                 <property name="text">
                  <string>0</string>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Parametres</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Up:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLineEdit" name="uplineEdit">
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Down:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="downlineEdit">
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Left:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLineEdit" name="leftlineEdit">
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Right:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="1">
                  <widget class="QLineEdit" name="rightlineEdit">
                   <property name="text">
                    <string>0</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0">
                  <widget class="QPushButton" name="confirmButton">
                   <property name="text">
                    <string>Confirm</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Information</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>X:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="xlineEdit">
               <property name="text">
                <string>0</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Y:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="ylineEdit">
               <property name="text">
                <string>0</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Depth:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="depthlineEdit">
               <property name="text">
                <string>0</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Distance:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="DislineEdit">
               <property name="text">
                <string>0</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Distance</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>MaxDepth:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="maxdepthLineEdit">
               <property name="text">
                <string>30000</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Mindepth:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="mindepthLineEdit">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QPushButton" name="setColorButton">
               <property name="text">
                <string>Setdepth</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="graylabel">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>240</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>10000</width>
            <height>240</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="colorlabel">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>240</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>10000</width>
            <height>10000</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>533</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="saveallAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="opentoolAction"/>
   <addaction name="savetoolAction"/>
   <addaction name="savealltoolAction"/>
   <addaction name="abouttoolAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="opentoolAction">
   <property name="icon">
    <iconset resource="fileconversion.qrc">
     <normaloff>:/FileConversion/open.ico</normaloff>:/FileConversion/open.ico</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="savetoolAction">
   <property name="icon">
    <iconset resource="fileconversion.qrc">
     <normaloff>:/FileConversion/Resources/save.ico</normaloff>:/FileConversion/Resources/save.ico</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="savealltoolAction">
   <property name="icon">
    <iconset resource="fileconversion.qrc">
     <normaloff>:/FileConversion/Resources/saveall.ico</normaloff>:/FileConversion/Resources/saveall.ico</iconset>
   </property>
   <property name="text">
    <string>SaveAll</string>
   </property>
  </action>
  <action name="abouttoolAction">
   <property name="icon">
    <iconset resource="fileconversion.qrc">
     <normaloff>:/FileConversion/Resources/help.ico</normaloff>:/FileConversion/Resources/help.ico</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="fileconversion.qrc">
     <normaloff>:/FileConversion/open.ico</normaloff>:/FileConversion/open.ico</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="fileconversion.qrc">
     <normaloff>:/FileConversion/Resources/save.ico</normaloff>:/FileConversion/Resources/save.ico</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="saveallAction">
   <property name="icon">
    <iconset resource="fileconversion.qrc">
     <normaloff>:/FileConversion/Resources/saveall.ico</normaloff>:/FileConversion/Resources/saveall.ico</iconset>
   </property>
   <property name="text">
    <string>SaveAll</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset resource="fileconversion.qrc">
     <normaloff>:/FileConversion/Resources/help.ico</normaloff>:/FileConversion/Resources/help.ico</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="fileconversion.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>backButton</sender>
   <signal>clicked()</signal>
   <receiver>FileConversionClass</receiver>
   <slot>TurnLeftSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>526</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>555</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextButton</sender>
   <signal>clicked()</signal>
   <receiver>FileConversionClass</receiver>
   <slot>TurnRightSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>527</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmButton</sender>
   <signal>clicked()</signal>
   <receiver>FileConversionClass</receiver>
   <slot>ConfirmSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ConfirmSlot()</slot>
  <slot>SetColorSlot()</slot>
  <slot>TurnSlot()</slot>
  <slot>TurnLeftSlot()</slot>
  <slot>TurnRightSlot()</slot>
 </slots>
</ui>
